static-web.css

```diff
@@ -0,0 +1,55 @@
+body{
+	width: 100%;
```

Comment:
No need to set width on the body like this. The body dimensions should be left untouched